armbian/base/scripts/prometheus-base.py

```diff
@@ -5,13 +5,19 @@
 import time
 import subprocess
 import sys
-from prometheus_client import start_http_server, Gauge, Counter
+from prometheus_client import start_http_server, Gauge, Counter, Info
+
+SYSCONFIG_PATH="/opt/shift/sysconfig/"
 
 # Create Prometheus metrics to track bitcoind stats.
```

Comment:
(Optional) Not new in this PR, but perhaps this comment would be more accurate if written as `Create Prometheus metrics to track Base stats.`?